example.js

```diff
@@ -105,13 +110,26 @@ define([
 		});
 	}
 	
-	function unmuteVideo() {
-		setMuteButton(false);
```

Comment:
This function is no longer necessary. See [here](https://github.com/mirainc/sdk-webrtc-meetings/pull/3/files#diff-65d7f35d2dc8f88273f9b1e3438d3af2L92).